libs/langchain/langchain/llms/gradient_ai.py

```diff
@@ -234,3 +248,34 @@ async def _acall(
             text = enforce_stop_tokens(text, stop)
 
         return text
+
+    def _generate(
```

Comment:
why do we need to implement _generate? parent class LLM already implements

Reply:
if we're implementing _generate then we should inherit from BaseLLM instead of LLM and don't need to implement _call